packages/frontend/admin/src/modules/settings/config-input-row.tsx

```
@@ -132,7 +132,10 @@ export const ConfigRow = ({
       className={`flex justify-between flex-grow space-y-[10px]
          ${type === 'Boolean' ? 'flex-row' : 'flex-col'}`}
     >
-      <div className="text-base font-bold flex-3">{desc}</div>
+      <div
+        className="text-base font-bold flex-3"
+        dangerouslySetInnerHTML={{ __html: desc }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)